docs/apireference/README.md

```diff
@@ -13,8 +13,11 @@ go get -u github.com/kubernetes-incubator/apiserver-builder/cmd/...
 # Install the reference docs commands (apiserver-builder commands invoke these)
 go get -u github.com/kubernetes-incubator/reference-docs/gen-apidocs/...
```

Comment:
We should also mention here that we expect `make gen-api-docs` to work, presuming we think it now works!

Reply:
Another PR?  We need to figure out the install instructions.  I installed this differently than you did ...

I had to install `lister-gen`. @pwittrock any ideas?


```
go install k8s.io/code-generator/cmd/lister-gen
```